HISTORY.md

```diff
@@ -113,3 +113,5 @@
   - Added `JOYP_SGB_*` constants
   - Added more `BOOTUP_*` value constants
   - Corrected comments on some audio registers
+- **Rev 5.4.0** - 2025-12-31 *(mattcurrie)*
+  - Added registers and constants for PSW
```

Comment:
We're no longer doing "one commit = one release", so this doesn't need to be here. There are pretty many issues and PRs open by now, and they'll get consolidated into a 5.4.0.
```suggestion
```